public/js/angular/controllers.js

```diff
@@ -1,6 +1,6 @@
 angular
-  .module('webmakerApp')
-  .controller('navigationController', ['$scope', '$location', '$routeParams', '$rootScope', 'weblit', 'wmNav', 'CONFIG',
+  .module("webmakerApp")
```

Comment:
I lean towards single quote but I'll use double if there are a lot already, or if there is a string in the file with a single quote as punctuation. Why escape it if we don't need to. *shrugs*

I didn't want to get into enforcing single or double, just as long as it's the same per file.